evm-test-helpers/src/setup.ts

```diff
@@ -43,7 +45,15 @@ export function provider(): ethers.providers.JsonRpcProvider {
     providerEngine.addProvider(revertTraceSubprovider)
   }
 
-  providerEngine.addProvider(new GanacheSubprovider({ gasLimit: 8_000_000 }))
+  const options = {
+    gasLimit: 8_000_000,
```

Comment:
It's higher now, right? Wondering if we should raise this to match mainnet. Probably out of scope for this PR, just noting it.

Reply:
Yes, we are at [`12_500_000` currently](https://etherscan.io/chart/gaslimit).